src/compilerWrapper.ts

```diff
@@ -96,6 +97,11 @@ export interface AutoTypedVar {
 	type: string;
 }
 
+export interface IntConstVar {
```

Comment:
maybe `ConstIntVar` for `const int x = 1;`

Reply:
removed